types/viz.js/tsconfig.json

```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
+            "dom",
```

Comment:
This is required to reference `SVGSVGElement` and `HTMLImageElement`. I'm not sure if this will cause problems for anyone trying to use `viz.js` from Node.

Reply:
These changes don't appear to negatively affect my node app using viz.js. It didn't need to add this lib entry to continue to function.